.github/workflows/build.yml

```diff
@@ -24,9 +24,30 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  get_ci_fixes:
```

Comment:
This adds now another row in the checks under each PR, which is already a pretty crowded. Can this not be merged directly in the build workflow as you do say with conda?

Reply:
No, because the container does not have a suitable version of the gh script available, and installing it is too annoying.

Reply:
@mkoeppe, No response to this?

Reply:
You may need to switch a different view; I responded above

Reply:
https://github.com/marketplace/actions/install-gh-cli or something similar?

Reply:
that one's not portable to i686, and overall I don't think this is a worthwhile change

Reply:
I don't like the duplicate lines too. But that's a trivial issue, and there's no simple fix. Let's ignore it.

Reply:
> We are not using any i686 architecture...that's a very potential if and when argument...

Yes, we are. The build.yml workflow accepts the container image on workflow_dispatch, and we do test 32-bit platforms.